node/pkg/common/chainlock.go

```diff
@@ -18,3 +21,82 @@ type MessagePublication struct {
 	EmitterAddress   vaa.Address
 	Payload          []byte
 }
+
```

Comment:
Would be nice to have unit-tests defined in chainlock_test.go for the new functions defined in chainlock.go